### ISSUE TITLE
Autoloader problem

### ISSUE THREAD

**Issue by soee** [yiiext/twig-renderer#12]
Trying to run Twig using installation steps described on github cause errors. It looks like autoloader function defined in Twig/Autoloader.php is not used at all. First line of error message is:
is_file(): open_basedir restriction in effect. File(/usr/share/php/Twig_Loader_Filesystem.php) is not within the allowed path(s): 

The workaround i have used was to comment out this two lines:

```
    require Yii::getPathOfAlias($this->twigPathAlias).'/Autoloader.php';
    Yii::registerAutoloader(array('Twig_Autoloader', 'autoload'), true);
```

in ETwigViewRenderer.php file and inside my index.php, after require_once($yii); added again:

require_once dirname(**FILE**) . '/protected/vendors/Twig/Autoloader.php';
Yii::registerAutoloader(array('Twig_Autoloader', 'autoload'), false);

and finally i can use Twig now. The problem was on both 1.12 and 1.13 Yii versions.

Am i doing something wrong or autoloader function has to be defined before application is created ?

**Comment by ErrorProne** [comment]
TL;DR:
Add include_path = "." to the php.ini

Long:
For others finding this. It is indeed a php.ini configuration issue, but not as it looks on the first look, that open_basedir is too strict. 

e.g. Ubuntu (tested with 16.04) doesn't per default have a /usr/share/php folder, however PHP (at least the apache module) ships with a default value that looks like this: 
_include_path = .:/usr/share/php_

The implementation of is_file seems to think that a folder that does not exist is a restricted access (Disclaimer: I didn't look at the actual implementation, it is just a wild guess).
So the ugly way to fix this would be to give open_baseidr access to /usr/share/php AND create the directory structure or the better way enhance your php.ini with  _include_path = "."_
